src/default-options.ts

```diff
@@ -26,6 +26,12 @@ export const defaultOptions = {
     // in bytes, with the default lease manager.
     maxOutstandingBytes: 100 * 1024 * 1024,
 
+    // The minimum length of time a message's lease will be extended by.
+    minAckDeadline: undefined,
```

Comment:
Should this have a default value? The Java library has the `minAckDeadline` default to 10 seconds

Reply:
The Node library doesn't seem to have a default, and I was trying to maintain the existing behaviour for non-EOS. This is the relevant bit:

https://github.com/googleapis/nodejs-pubsub/blob/b255af63bdb150ccf42d90e2c4d17e6e94082ae9/src/lease-manager.ts#L278

Reply:
Oh, also to be clear - the Node library defaults ackDeadline to 10s, but if the user overrides it, that value is always used. There's no lower or upper bound on ackDeadline as it adjusts for 99th percentile. I gather Node is a bit of an outlier in this...